stdlib/uint32.gr

```diff
@@ -0,0 +1,411 @@
+/**
+ * @module Uint32: Unsigned utilities for working with the Int32 type.
+ * @example import Uint32 from "uint32"
+ *
+ * @since v0.5.0
+ */
+import WasmI32 from "runtime/unsafe/wasmi32"
+import Memory from "runtime/unsafe/memory"
+import Int32 from "int32"
+
+import {
+  coerceNumberToInt32 as fromNumber,
+  coerceInt32ToNumber as toNumber,
+} from "runtime/numbers"
+
+/**
+ * @section Conversions: Functions for converting between Numbers and the Int32 type.
+ */
+
+/**
+ * Converts a Number to an Int32.
+ *
+ * @param number: The value to convert
+ * @returns The Number represented as an Int32
+ *
+ * @since v0.5.0
+ */
+export fromNumber
+
+/**
+ * Converts an Int32 to a Number.
+ *
+ * @param value: The value to convert
+ * @returns The Int32 represented as a Number
+ *
+ * @since v0.5.0
+ */
+export toNumber
+
+/**
+ * @section Operations: Mathematical operations for Int32 values.
+ */
+
+/**
+ * Increments the value by one.
+ *
+ * @param value: The value to increment
+ * @returns The incremented value
+ *
+ * @since v0.5.0
+ */
+export let incr = Int32.incr
+
+/**
+ * Decrements the value by one.
+ *
+ * @param value: The value to decrement
+ * @returns The decremented value
+ *
+ * @since v0.5.0
+ */
+export let decr = Int32.decr
+
+/**
+ * Computes the sum of its operands.
+ *
+ * @param x: The first operand
+ * @param y: The second operand
+ * @returns The sum of the two operands
+ *
+ * @since v0.5.0
+ */
+export let add = Int32.add
+
+/**
+ * Computes the difference of its operands.
+ *
+ * @param x: The first operand
+ * @param y: The second operand
+ * @returns The difference of the two operands
+ *
+ * @since v0.5.0
+ */
+export let sub = Int32.sub
+
+/**
+ * Computes the product of its operands.
+ *
+ * @param x: The first operand
+ * @param y: The second operand
+ * @returns The product of the two operands
+ *
+ * @since v0.5.0
+ */
+export let mul = Int32.mul
+
+/**
+ * Computes the quotient of its operands using unsigned division.
+ *
+ * @param x: The first operand
+ * @param y: The second operand
+ * @returns The quotient of its operands
+ *
+ * @since v0.5.0
+ */
+export let div = Int32.divU
+
+/**
+ * Computes the quotient of its operands using signed division.
+ *
+ * @param x: The first operand
+ * @param y: The second operand
+ * @returns The quotient of its operands
+ *
+ * @since v0.5.0
+ */
+export let divS = Int32.div
```

Comment:
@ospencer was right, this feels really weird to me. Just re-exporting with the names swapped doesn't make sense to me when we already allow people to "import-as"